windows/RNSVG/RenderableView.cpp

```diff
@@ -222,7 +222,7 @@ void RenderableView::Render(
     strokeStyle.LineJoin(StrokeLineJoin());
     strokeStyle.DashOffset(StrokeDashOffset());
     strokeStyle.MiterLimit(StrokeMiterLimit());
-    strokeStyle.CustomDashStyle(Utils::GetValueArray(StrokeDashArray()));
+    strokeStyle.CustomDashStyle(Utils::GetAdjustedStrokeArray(StrokeDashArray(), StrokeWidth()));
```

Comment:
Pass in the adjusted strokeWidth as a float.

float strokeWidth{Utils::GetSvgLengthValue(StrokeWidth(), canvas.Size().Width)};